plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformUtils2.kt

```diff
@@ -1,14 +0,0 @@
-// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Main fix: remove duplicate util files to stop circular dependency issue. Combine into a single shared util file.